packages/webmonetization-polyfill-utils/src/lib/MonetizationTagObserver.ts

```diff
@@ -82,7 +82,13 @@ export class MonetizationTagObserver {
     const metas: MetaList = this.head.querySelectorAll(
       'meta[name="monetization"],link[rel="monetization"]'
     )
-    metas.forEach(this.onAddedTag.bind(this))
+    metas.forEach(m => {
+      try {
+        this.onAddedTag(m)
```

Comment:
https://github.com/coilhq/web-monetization-projects/blob/d3c533c84d5565875432fae5db2894b3f4a3305e/packages/coil-extension/src/content/services/ContentScript.ts#L205-L214

Prior to this change (swallowing exceptions and simply logging them, which is essentially what is done when tags are detected dynamically) if there are 2 tags in the DOM already when the above code runs, then the pause/resume handlers will not be set, thus allowing background monetization.